packages/core/src/storages/utils.ts

```diff
@@ -16,10 +16,33 @@ import { Configuration } from '../configuration';
  * this method will make sure the storage is purged only once for a given execution context, so it is safe to call
  * it multiple times.
  */
```

Comment:
this jsdoc belongs to the method. and since you used function overloading, it needs to be on all of the signatures (ideally you should have a different description based on the signature).